src/Http/Routing/src/DataSourceDependentCache.cs

```diff
@@ -8,6 +8,8 @@
 
 namespace Microsoft.AspNetCore.Routing
 {
+    // FYI: This class is also linked into MVC. If you make changes to the API you will
```

Comment:
You kinda don't need this comment per se. The build would fail if you #yolo change this.

Reply:
meh.